ocaml/parsing/depend.ml

```diff
@@ -605,6 +569,8 @@ and add_module_expr bv modl =
       add_expr bv e
   | Pmod_extension e ->
       handle_extension e
+  (* CR nroberts for lmaurer: what should happen here?  *)
+  | Pmod_instance _ -> ()
```

Comment:
@lukemaurer How should we be treating module instances in ocamldep?

Reply:
Luke tells me we should add a dependency on all module names that appear in the instance.